<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>CGAL Constrained Delaunay Triangulation</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/Logos.qrc" >:/cgal/logos/cgal_logo.xpm</iconset>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QGraphicsView" name="graphicsView" />
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="separator" />
    <addaction name="actionClear" />
    <addaction name="actionLoadConstraints" />
    <addaction name="actionSaveConstraints" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="title" >
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionInsertRandomPoints" />
   </widget>
   <widget class="QMenu" name="menuTools" >
    <property name="title" >
     <string>&amp;Tools</string>
    </property>
    <addaction name="actionMovingPoint" />
    <addaction name="actionInsertPolyline" />
    <addaction name="actionShowDelaunay" />
    <addaction name="actionShowVoronoi" />
   </widget>
   <widget class="QMenu" name="menuOptions" >
    <property name="title" >
     <string>&amp;Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout" />
    <addaction name="actionAboutCGAL" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuEdit" />
   <addaction name="menuTools" />
   <addaction name="menuOptions" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="fileToolBar" >
   <property name="windowTitle" >
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear" />
   <addaction name="actionLoadConstraints" />
   <addaction name="actionSaveConstraints" />
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionInsertPolyline" />
   <addaction name="actionMovingPoint" />
   <addaction name="actionShowDelaunay" />
   <addaction name="actionShowVoronoi" />
   <addaction name="actionCircumcenter" />
  </widget>
  <action name="actionAbout" >
   <property name="text" >
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL" >
   <property name="text" >
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="text" >
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionInsertRandomPoints" >
   <property name="text" >
    <string>&amp;Insert random points</string>
   </property>
  </action>
  <action name="actionMovingPoint" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="MainWindow.qrc" >:/cgal/Actions/icons/movingPoint.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Moving Point</string>
   </property>
   <property name="toolTip" >
    <string comment="The comment" >Simulate Insertion</string>
   </property>
   <property name="statusTip" >
    <string comment="and its comment" >Move mouse with left button pressed to see where point would be inserted</string>
   </property>
   <property name="whatsThis" >
    <string comment="what" >whats this</string>
   </property>
  </action>
  <action name="actionInsertPolyline" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Input.qrc" >:/cgal/Input/inputPolyline.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Insert Polyline</string>
   </property>
   <property name="toolTip" >
    <string>Insert Point or Polyline</string>
   </property>
   <property name="statusTip" >
    <string>Left: Insert vtx | Right: Final vtx | Del: Delete vtx</string>
   </property>
  </action>
  <action name="actionClear" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Clear</string>
   </property>
  </action>
  <action name="actionShowVoronoi" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="checked" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Triangulation_2.qrc" >:/cgal/Triangulation_2/Voronoi_diagram_2.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Voronoi Diagram</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionShowDelaunay" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../icons/Triangulation_2.qrc" >:/cgal/Triangulation_2/Delaunay_triangulation_2.png</iconset>
   </property>
   <property name="text" >
    <string>Show &amp;Delaunay Triangulation</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionLoadConstraints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Load Constraints</string>
   </property>
  </action>
  <action name="actionSaveConstraints" >
   <property name="icon" >
    <iconset resource="../icons/File.qrc" >:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Save Constraints</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionCircumcenter" >
   <property name="checkable" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="MainWindow.qrc" >:/cgal/Actions/icons/circumcenter.png</iconset>
   </property>
   <property name="text" >
    <string>&amp;Circumcenter</string>
   </property>
   <property name="toolTip" >
    <string>Draw circumcenter</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="MainWindow.qrc" />
  <include location="../icons/File.qrc" />
  <include location="../icons/Logos.qrc" />
  <include location="../icons/Triangulation_2.qrc" />
  <include location="../icons/Input.qrc" />
 </resources>
 <connections/>
</ui>
